pkg/cloudprovider/aws/instancetypes.go

```diff
@@ -34,11 +34,11 @@ import (
 )
 
 const (
-	InstanceTypesCacheKey                         = "types"
-	InstanceTypeZonesCacheKey                     = "zones"
-	InstanceTypesAndZonesCacheTTL                 = 5 * time.Minute
-	InsufficientCapacityErrorCacheTTL             = 45 * time.Second
-	InsufficientCapacityErrorCacheCleanupInterval = 5 * time.Minute
+	InstanceTypesCacheKey                          = "types"
+	InstanceTypeZonesCacheKey                      = "zones"
+	InstanceTypesAndZonesCacheTTL                  = 5 * time.Minute
+	UnfulfillableCapacityErrorCacheTTL             = 1 * time.Minute
+	UnfulfillableCapacityErrorCacheCleanupInterval = 5 * time.Minute
```

Comment:
Is 1 minute still too short?  I don't have any frame of reference for what good values would be, but maybe something like 5 and 10 minutes?  1 minute seems a little fast for anything to change with respect to capacity.

Reply:
I upped to 3-min since I still want it to be fairly aggressive on retrying. I removed the separate clean-up interval and will use the default one that the rest of the caches use, which is 10-min.